src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -119,4 +142,26 @@ namespace TerminalAppLocalTests
         VERIFY_SUCCEEDED(result);
     }
 
+    void TabTests::CreateSimpleTerminalXamlType()
+    {
+        winrt::com_ptr<winrt::TerminalApp::implementation::MinMaxCloseControl> mmcc{ nullptr };
+
+        auto result = RunOnUIThread([&mmcc]() {

```

Comment:
Can we not use the co_await pattern here like #4051?

Reply:
I don't know. RunOnUIThread was already cribbed from TAEF or something, but they didn't really have cppwinrt. Maybe that would work? I'll try

Reply:
Yea I don't think that'll work any nicer. All those were able to be called from `fire_and_forget` functions. However, the TAEF `TEST_METHOD` has some other return type, that we can't use `co_await` with.